pairing_bot.go

```diff
@@ -49,9 +47,10 @@ type PairingLogic struct {
 	pdb   PairingsDB
 	revdb ReviewDB
 
-	zulip   *zulip.Client
-	recurse *recurse.Client
-	version string
+	zulip           *zulip.Client
+	recurse         *recurse.Client
+	version         string
+	maintenanceMode bool
```

Comment:
Will need to make export this eventually if/when this moves into its own package.